x-pack/legacy/plugins/uptime/server/lib/helper/get_histogram_interval.ts

```diff
@@ -7,13 +7,27 @@
 import DateMath from '@elastic/datemath';
 import { QUERY } from '../../../common/constants';
 
+export const parseRelativeEndDate = (_dateEnd: string) => {
+  // We need this this parsing because if user selects This week or this date
+  // That represents end date in future, if week or day is still in the middle
+  // Uptime data can never be collected in future, so we will reset end date to now
+  // in That case. Example case we select this week range will be to='now/w' and from = 'now/w'debugger;
```

Comment:
s/debugger//

Reply:
i was using webstorm debugger, it works pretty well :)

Reply:
done